addons/binding/org.openhab.binding.lgwebos/.classpath

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con"
+		path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8" />
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins" />
+	<classpathentry kind="src" path="src/main/java" />
+	<classpathentry kind="src" path="src/main/resources" />
+	<classpathentry kind="lib" path="lib/java-websocket-1.3.2.jar" />
+	<classpathentry kind="lib" path="lib/json-20140107.jar" />
+	<classpathentry kind="lib" path="lib/xpp3-1.1.4c.jar" />
+	<classpathentry kind="lib" path="lib/Connect-SDK-Java-Core-1.0.jar" />
```

Comment:
Since I don't own all the devices that this documentation claims are supported I cannot test or troubleshoot it.
I only own an Apple TV and it did not work at all. 
Therefore I renamed the binding from initial connectsdk to lgwebos. a) connectsdk is no longer maintained b) users don't care what the underlying technology is, but what device it supports.

Reply:
Pitty I was hoping for the support for my Apple TV, what version do you have the one with or without apps?

Reply:
it is the older version and when lg released all connectsdk developers the new appletv was not yet released.